docs/user-guide/featured.md

```diff
@@ -0,0 +1,29 @@
+## Featured Notebooks & Collections
+
+```eval_rst
+.. meta::
+   :description: Featured Notebooks and Collections.
+```
+
+<a href="https://jovian.ai?utm_source=docs" target=_blank> Jovian </a> provides you with the feature to highlight some of your Notebooks & Collections on profile overview. You can feature like those
+related to your project , or some of your favourite topics ,or frameworks/languages used, or other important qualities that you find useful on your profile.
+
+### How to add Notebooks/Collections in Featured Section?
+
+1. For each notebook and collection you have been provided with the option to Pin it.
+   Note that you cannot pin private/secret notebooks and private collections.
+
+  <img src="https://i.imgur.com/oxzHbMc.png" class="screenshot" alt="Pin Notebook">
+
+  <img src="https://i.imgur.com/5k71Moy.png" class="screenshot" alt="Pin Collection">
+
+2. Navigate to Overview section after you have pinned the notebook/collection.There you see
+   your featured Notebooks and Collections.
+   Similarly you can see the featured collection and notebooks for other users as well.
```

Comment:
Notebook/Collection is different in each instance like
- collection and notebook
- notebook & collection
- notebook or collection

Please go over this PR and make it consistent

Reply:
Made the same instance everywhere.